runtime/gc_glue_java/HeapWalkerDelegate.cpp

```diff
@@ -37,7 +37,7 @@ MM_HeapWalkerDelegate::objectSlotsDo(OMR_VMThread *omrVMThread, omrobjectptr_t o
 	MM_EnvironmentBase *env = MM_EnvironmentBase::getEnvironment(omrVMThread);
 	switch(_objectModel->getScanType(objectPtr)) {
 	case GC_ObjectModel::SCAN_CONTINUATION_OBJECT:
-		doContinuationObject(env, objectPtr, function, userData);
+		scanContinuationNativeSlots(env, objectPtr, function, userData);
```

Comment:
we should be consistent in naming with the caller. Either the caller should be objectSlotsScan or this should be doContinuationNativeSlots. The latter is probably less intrusive change (other Walker APIs also tend to use 'do')